NeuroID/NeuroIDClass/Extensions/NIDSend.swift

```diff
@@ -50,6 +50,39 @@ internal extension NeuroID {
         return workItem
     }
 
+    static func initGyroAccelCollectionTimer() {
+        if let workItem = NeuroID.sendGyroAccelCollectionWorkItem {
+            // Send up the first payload, and then setup a repeating timer
+            DispatchQueue
+                .global(qos: .utility)
+                .asyncAfter(
+                    deadline: .now() + GYRO_SAMPLE_INTERVAL, // 200 ms
```

Comment:
Does the .now() return time in seconds or nanoseconds? Wasn't able to tell from the docs.

Reply:
Looks like .now() is in nanoseconds but adding seconds to it in here seems to be working as expected.